app/services/idv/actions/verify_document_status_action.rb

```diff
@@ -65,10 +65,23 @@ def document_capture_session
       end
 
       def async_state
-        return timed_out if document_capture_session.nil?
+        if document_capture_session.nil?
+          @flow.analytics.track_event(Analytics::DOC_AUTH,
```

Comment:
not very familiar with analytics, should it be a new key here?

Reply:
Probably better for it to be its own key so it's easier for us to identity these debug events vs the normal doc auth ones

Reply:
Sounds good, updated in 00c5d8a